src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/config.jelly

```diff
@@ -17,5 +17,11 @@
         <f:entry title="${%Checkout credentials}" field="checkoutCredentialsId">
             <c:select default="${descriptor.SAME}"/>
         </f:entry>
+            <f:entry title="${%Include branches}" field="includes">
+        <f:textbox default="${descriptor.defaultIncludes}"/>
```

Comment:
:bug: `descriptor` here is a `GitHubSCMNavigator.DescriptorImpl` so you would need to either refer to the `GitHubSCMSource.DescriptorImpl` or (more easily) define constants in `GitHubSCMNavigator.DescriptorImpl` mirroring those in `GitHubSCMSource.DescriptorImpl`.